### PR TITLE
Feat/admin merchant create

### LINKED ISSUE #31: Admin Merchant Create
Admin Merchant Create

As an admin,
When I visit the admin merchants index
I see a link to create a new merchant.
When I click on the link,
I am taken to a form that allows me to add merchant information.
When I fill out the form I click ‘Submit’
Then I am taken back to the admin merchants index page
And I see the merchant I just created displayed
And I see my merchant was created with a default status of disabled.

### PR DESCRIPTION
**Pull request by Dominicod**
**Fixes large bug in seeds**
* Create tests for Create Merchant
* Create logic for Create Merchant
* Reorganize test specs to correct files
* Fix bug in rake tasks to fix PK sequence not being reinitialized

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
   #get '/merchants/:id/items/:id', to: 'items#edit'
   resources :admin, only: [:index]
   namespace :admin do
-    resources :invoices, only: [:index, :show, :update]
-    resources :merchants, only: [:index, :show, :edit, :update]
+    resources :invoices, only: [:index, :show]
+    resources :merchants, only: [:index, :show, :edit, :update, :new, :create]
```
Comment (thayes87) [inline config/routes.rb:12]: I think at this point you can remove the only as all the actions have now been added.

File: spec/features/admin/merchants/edit_spec.rb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      end
+
+      expect(page.current_path).to eq admin_merchant_path(1)
+      expect(page).to have_content("test_merchant")
+      expect(page).to_not have_content("Schroeder-Jerde")
+    end
+    end
+
+    it "I see a flash message stating that the information has been successfully updated." do
+      visit admin_merchants_path
```
Comment (thayes87) [inline spec/features/admin/merchants/edit_spec.rb:49]: Probably need to add some sad path testing (I also need to add on the Merchants Items Side)

**Review by thayes87 — APPROVED** [review]
Great catch with the lower case table names

File: spec/features/admin/merchants/edit_spec.rb (hunk 2 above)
Comment (Dominicod) [reply spec/features/admin/merchants/edit_spec.rb:49]: Let me do this now!

File: config/routes.rb (hunk 1 above)
Comment (Dominicod) [reply config/routes.rb:12]: Let me do this now!

**Pull request closed without merging**